bindings/node.js/kzg.cxx

```diff
@@ -149,21 +127,20 @@ Napi::Value BlobToKzgCommitment(const Napi::CallbackInfo& info) {
     return env.Undefined();
   };
 
-  return napi_typed_array_from_bytes((uint8_t *)(&commitment), BYTES_PER_COMMITMENT, env);
+  return Napi::Buffer<uint8_t>::Copy(env, reinterpret_cast<uint8_t *>(&commitment), BYTES_PER_COMMITMENT);
```

Comment:
That's much simpler! Is there any weird way that Buffer::Copy might fail that we should be aware of?

Reply:
Yep.  It's the exact purpose that function was built.  There are not a lot of difference between the `Buffer<uint8_t>` and the underlying `TypedArray<uint8_t>` aside from the `Copy` method.  On the JS side the Buffer object has a couple other features that are nice to haves (like being able to choose encodings when calling `toString('hex')` vs `toString()`). Both have the method but `Buffer` allows passing the encoding, in particular for serialization as a `hex` encoded string.

As far as failure methods, that I am not sure about honestly.  The node repo is one of the most heavily tested and it has huge visibility so stability is key but that is most definitely the answer that you would want to hear.  I did as much googling as I could to try and find search term combinations that would provide some insight but it was not very fruitful.  I also looked through the old node and node-addon-api issues and didnt find much either.

[here](https://github.com/nodejs/node-addon-api/blob/0b53d885f50c83cd7cbcf505d3dcc89cf8c3d59f/test/buffer.cc#L108) is the unit test for that function but i didnt see any failure tests